components/App.jsx

```diff
@@ -3,8 +3,12 @@ import Paper from 'material-ui/Paper';
 import AppBar from 'material-ui/AppBar';
 import SearchBox from '../containers/SearchBox';
 import MovieDetail from '../containers/MovieDetail';
+
```

Comment:
Nitpick: Pls remove extra whitespaces to clean up formatting. (not worth fixing now, but something to keep in mind to maintain code.)